tests/foreman/cli/test_satellitesync.py

```diff
@@ -232,20 +382,30 @@ def class_export_entities(default_sat):
 
 
 @pytest.fixture(scope='function')
-def function_export_cv_directory(default_sat):
-    """Create directory for CV export and at the end remove it"""
-    export_dir = f'{get_export_base()}/{gen_string("alpha")}'
-    default_sat.execute(f'mkdir {export_dir}')
-    yield export_dir
+def export_cleanup_library(default_sat, function_org):
+    """Deletes dir for library export"""
+    yield
+    # Deletes directory created for library export test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}').stdout == ''
+
+
+@pytest.fixture(scope='function')
+def export_cleanup(default_sat, module_org):
+    """Deletes dir for version export"""
+    yield
     # Deletes directory created for CV export Test
-    default_sat.execute(f'rm -rf {export_dir}')
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{module_org.name}').stdout == ''
 
 
-def get_export_base():
-    return '/var/lib/pulp/katello-export'
+def validate_filepath(default_sat):
+    """Checks the existence of certain files in a dir"""
+    result = default_sat.execute(
```

Comment:
i'd recommend changing the name from "default_sat" to something that doesn't match a fixture name. this will make it less confusing for those that might think this is accepting the default_sat fixture. this would also be something a user could use against another non-default satellite object.